src/screens/EventDashboard/EventDashboard.tsx

```diff
@@ -77,10 +34,9 @@ const EventDashboard = (): JSX.Element => {
     <LeftDrawerEventWrapper
       event={eventData.event}
       key={`${eventData?.event._id || 'loading'}EventDashboard`}
-      setShowAddEventProjectModal={setShowAddEventProjectModal}
     >
       <Row>
-        <Col sm={4}>
+        <Col>
```

Comment:
Why is this changed?

Reply:
The row had 2 columns and after removing the second col (related to event projects), I removed the fixed width of the first one which gave it the full width in the row as it's now the only col in that row. It doesn't change the UI